src/ado/reprun.ado

```diff
@@ -6,6 +6,9 @@ cap program drop   reprun
 qui {
 
     version 14.1
+	
+    timer clear 1
+    timer on 1
```

Comment:
I would suggest using a higher number -- if the user is using a timer, they may use `1` and this will cause an error or other collision.